apps/web/app/globals.scss

```
@@ -1,5 +1,5 @@
 /* Token source of truth: packages/shared/src/tokens.ts */
-@import "tailwindcss";
+@import url("tailwindcss");
```

Stylelint (17.6.0)

[error] Expected "url("tailwindcss")" to be ""tailwindcss"" (import-notation)

(import-notation)